week2/aniketsaoji/fizzbuzz.java

```diff
@@ -0,0 +1,28 @@
+import java.util.*;
+
+public class fizzbuzz {
+
+	public static int returnRandom() {
+		Random rand = new Random();
+		return rand.nextInt(100) + 1;	
+	}
+
+	public static void main (String [] args) {
+		int count = 1;
+		int randomNumber = returnRandom();
+		while (count < 101) {
+			while (randomNumber != count) {
+				randomNumber = returnRandom();
```

Comment:
This should be deterministic, i.e. should always print out numbers 1-100, not random numbers